src/socketio.quiver.js

```diff
@@ -40,6 +43,19 @@ function QuiverSocialProvider(dispatchEvent) {
   this.dispatchEvent = dispatchEvent;
   this.storage = freedom['core.storage']();
 
+  /** @private @const {!FreedomCrypto} */
+  this.crypto_ = freedom['core.crypto']();
+
+  /** @private @const {!FreedomPgp} */
+  this.pgp_ = freedom['pgp-e2e']();
+
+  /** @type {!Promise<string>} */
+  this.getPubKey_ = this.pgp_.setup('', '<quiver>').then(function() {
```

Comment:
I have a branch of uProxy that removes encryption for Quiver in the uProxy layer and passes in a new ```pgpKeyName``` parameter as one of the login params: https://github.com/uProxy/uproxy/compare/dborkan-remove-quiver-encryption?expand=1